sdk/deviceupdate/Azure.IoT.DeviceUpdate/samples/DeviceUpdateClientSample/Sample.cs

```diff
@@ -2,14 +2,15 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Linq;
```

Comment:
It is not required for this preview, but before GA, please use the standard format for samples in this repo.  A good example is shown here: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/search/Azure.Search.Documents/samples, and the official guidance is written out here: https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples

Reply:
The issue to track: https://github.com/Azure/azure-sdk-for-net/issues/26455